homeassistant/components/nissan_leaf/__init__.py

```diff
@@ -85,25 +85,22 @@
 })
 
 
-async def async_setup(hass, config):
+def setup(hass, config):
     """Set up the Nissan Leaf component."""
     import pycarwings2
 
-    async def handle_update(service):
+    async def async_handle_update(service):
         # It would be better if this was changed to use nickname, or
         # an entity name rather than a vin.
         vin = service.data.get(ATTR_VIN, '')
```

Comment:
Use `dict[key]` for required schema keys and keys with default schema values.

Reply:
Fixed with commit https://github.com/home-assistant/home-assistant/pull/21145/commits/2b610fa7c47fc0746de091996791b91bee617b09